xds/README.md

```diff
@@ -7,115 +7,153 @@ upstream endpoints and filter configuration from some external source, and then
 that information to generate cluster and filter updates that is pushed to any connected
 Quilkin proxy.
 
-The project has two binaries depending on the external source of configuration:
-
-1. **cmd/file/file.go** an implementation that watches a configuration file on disk and
-   sends updates to proxies whenever that file changes.
-   It can be started with the following command:
-   ```sh
-   go run cmd/file/file.go --config=config.yaml --port=18000
-   ```
-   After running this command, any proxy that connects to port 18000 will receive updates as
-   configured in `config.yaml` file.
-   The configuration file schema is:
-   ```yaml
-   # clusters contain a list of clusters.
-   # Each entry represents a cluster configuration.
-   clusters: [{
-     # Name of the cluster.
-     name: string
-
-     # List of endpoints belonging to the cluster.
-     # Each entry represents an upstream endpoint.
-     endpoints: [{
-       # The endpoint's IP address.
-       ip: int
-       # The endpoint's port.
-       port: int
-       # Opaque metadata that will be the endpoint's metadata.
-       metadata: {}
-     }]
-   }]
-
-   # filterchain represents the filter chain configuration.
-   # It contains a list of filter configurations.
-   filterchain: [{
-     # Name of the filter
-     name: string
-
-     # typed_config contains the filter's configuration.
-     typed_config: {
-       # @type must be equivalent to name - the name of the filter.
-       # It is an extra, required field.
-       '@type': string
-       # ...
-       # The rest of the body contains filter specific configuration or
-       # is empty if the filter has no configuration.
-     }
-   }]
-   ```
-   Example:
-   ```yaml
-   clusters:
-   - name: cluster-a
-     endpoints:
-     - ip: 123.0.0.1
-       port": 29
-       metadata:
-         'quilkin.dev':
-            tokens:
-            - "MXg3aWp5Ng=="
-   filterchain:
-   - name: quilkin.extensions.filters.debug.v1alpha1.Debug
-     typed_config:
-       '@type': quilkin.extensions.filters.debug.v1alpha1.Debug
-       id: hello
-   ```
-
-   > The file.go binary is primarily an example and mostly suitable for demo purposes.
-   > As a result, some configuration options and features might be missing.
-
-1. **cmd/controller.go**: A server implementation that runs in [Kubernetes].
+The project has a binary **cmd/controller.go** with a server implementation that runs in [Kubernetes].
```

Comment:
Want to add some big headers, finding it hard to separate the sections when looking over all the text (also will give us a nice TOC).

```suggestion
## Opinionated Agones Implementation

The project has a binary **cmd/controller.go** with a server implementation that runs in [Kubernetes].
```

Reply:
Added!